addons/godot-firebase/firebase/firebase.gd

```diff
@@ -65,8 +65,11 @@ var _config : Dictionary = {
             "functions" : "",
             "storage" : "",
             "dynamicLinks" : ""
-           }
-       }
+        }
+    },
+    "workarounds":{
+        "database_connection_closed_issue": false, # fixes https://github.com/firebase/firebase-tools/issues/3329
+    }
```

Comment:
Added a flag for the workaround disabling it by default since it only effects emulation
@WolfgangSenff